dev/WebView2/WebView2.cpp

```diff
@@ -852,80 +852,47 @@ winrt::Size WebView2::ArrangeOverride(winrt::Size const& finalSize)
 void WebView2::FillPointerPenInfo(const winrt::PointerPoint& inputPt, winrt::CoreWebView2PointerInfo outputPt)
 {
     winrt::PointerPointProperties inputProperties{ inputPt.Properties() };
-    UINT32 outputPt_penFlags{PEN_FLAG_NONE};
 
+    UINT32 outputPt_penFlags{ PEN_FLAG_NONE };
     if (inputProperties.IsBarrelButtonPressed())
     {
         outputPt_penFlags |= PEN_FLAG_BARREL;
     }
-
     if (inputProperties.IsInverted())
     {
         outputPt_penFlags |= PEN_FLAG_INVERTED;
     }
-
     if (inputProperties.IsEraser())
     {
         outputPt_penFlags |= PEN_FLAG_ERASER;
     }
-
     outputPt.PenFlags(outputPt_penFlags);
 
-    UINT32 outputPt_penMask = PEN_MASK_PRESSURE | PEN_MASK_ROTATION | PEN_MASK_TILT_X | PEN_MASK_TILT_Y;
-    outputPt.PenMask(outputPt_penMask);
-
-    UINT32 outputPt_penPressure = static_cast<UINT32>(inputProperties.Pressure()* 1024);
-    outputPt.PenPressure(outputPt_penPressure);
-
-    UINT32 outputPt_penRotation = static_cast<UINT32>(inputProperties.Twist());
-    outputPt.PenRotation(outputPt_penRotation);
-
-    INT32 outputPt_penTiltX = static_cast<INT32>(inputProperties.XTilt());
-    outputPt.PenTiltX(outputPt_penTiltX);
-
-    INT32 outputPt_penTiltY = static_cast<INT32>(inputProperties.YTilt());
-    outputPt.PenTiltY(outputPt_penTiltY);
+    outputPt.PenMask(PEN_MASK_PRESSURE | PEN_MASK_ROTATION | PEN_MASK_TILT_X | PEN_MASK_TILT_Y);
+    outputPt.PenPressure(static_cast<uint32_t>(inputProperties.Pressure() * 1024));
```

Comment:
>uint32_t [](http://example.com/codeflow?start=37&length=8)

UINT32 is the Win32 type expected in CoreWebView2PointerInfo, not sure if there is a reason to prefer uint32_t?

Reply:
The winrt projections are expecting the standard types (ie uint32_t) and that seemed cleaner.